Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -1871,9 +1871,17 @@ void MainWindow::hotkeyToggle(HotKeyID id, bool active)
 #endif
         break;
     case HOTKEY_VOICEACTIVATION :
-        if(active)
+        if(active) {
             TT_EnableVoiceActivation(ttInst, 

```

Comment:
Minor thing (I'll change it); In Qt the default code style is curly brace on next line.